tests/src/test/scala/scalaz/ValidationTest.scala

```diff
@@ -77,12 +78,12 @@ object ValidationTest extends SpecLite {
     "return first success" in {
       val succ = Validation.success[String, Int](1).toValidationNel
       val fail = Validation.failure[String, Int]("2").toValidationNel
-    
```

Comment:
@relrod I personally don't think it's a good idea in term of history quality.

If most contributors have a hook in their editor to do it when saving a file we will "fix" the whole codebase after a while naturally.